docs/minio-client-complete-guide.md

```diff
@@ -322,11 +323,11 @@ USAGE:
    mc cat [FLAGS] SOURCE [SOURCE...]
 
 FLAGS:
-  --help, -h                       Show help.
-  --encrypt-key value              Decrypt object (using server-side encryption)
+  --encrypt-key value           encrypt/decrypt objects (using server-side encryption customer provided keys)
```

Comment:
instead of 
> encrypt/decrypt objects (using server-side encryption customer provided keys

``` decrypt objects (using server-side encryption with customer provided keys)```

Reply:
The command comes from a common flag description will be same.